src/invidious/views/template.ecr

```diff
@@ -153,6 +153,11 @@
                         <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
                     </div>
                 </div>
+                <% if CONFIG.footer %>
+                    <div class="h-box v-box" id="footer-info">
```

Comment:
I don't know if this is intentional or not but if you'd like to preserve the bold text seen in the initial version feel free to add custom styling in default.css.  

```css
#footer-info {
  font-weight: bold;
}
```

Reply:
I decided at first that the bold text would be ok. Then I decided that the user should make this choice if they want it to through HTML. Thank you for the tip though :3